python/ray/train/data_parallel_trainer.py

```diff
@@ -373,6 +375,97 @@ def get_dataset_config(self) -> Dict[str, DatasetConfig]:
+            logger.warn(
+                "'ipywidgets' isn't installed. Run `pip install ipywidgets` to "
+                "enable notebook widgets."
+            )
+            return None
+
+        from IPython.display import display
+
+        title = HTML(f"<h2>{self.__class__.__name__}</h2>")
+
+        tab = Tab()
+        children = []
+
+        tab.set_title(0, "Train Loop Config")
+        children.append(
+            self._train_loop_config_repr_html_() if self._train_loop_config else None
+        )
+
+        tab.set_title(1, "Backend Config")
+        children.append(
+            self._backend_config._repr_html_() if self._backend_config else None
+        )
+
+        tab.set_title(2, "Dataset Config")
+        children.append(
+            self._dataset_config_repr_html_() if self._dataset_config else None
+        )
+
+        tab.set_title(3, "Scaling Config")
+        children.append(
+            self.scaling_config._repr_html_() if self.scaling_config else None
+        )
+
+        tab.set_title(4, "Run Config")
+        children.append(self.run_config._repr_html_() if self.run_config else None)
+
+        tab.set_title(5, "Datasets")
+        children.append(self._datasets_repr_html_() if self.datasets else None)
+
+        tab.children = [HTML(child) for child in children]
+        display(VBox([title, tab], layout=Layout(width="100%")))
+
+    def _train_loop_config_repr_html_(self) -> str:
+        content = []
+        if self._train_loop_config:
+            table_data = {}
+            for k, v in self._train_loop_config.items():
+                if isinstance(v, str) or str(v).isnumeric():
+                    table_data[k] = v
+                elif hasattr(v, "_repr_html_"):
+                    table_data[k] = v._repr_html_()
+                else:
+                    table_data[k] = str(v)
+
+            content = Template("title_data.html.j2").render(
+                title="Train Loop Config",
+                data=tabulate(
+                    table_data.items(),
+                    headers=["Setting", "Value"],
+                    showindex=False,
+                    tablefmt="unsafehtml",
+                ),
+            )
+        return Template("rendered_html_common.html.j2").render(content=content)
+
+    def _dataset_config_repr_html_(self) -> str:
+        content = []
+        if self._dataset_config:
+            for name, config in self._dataset_config.items():
+                content.append(
+                    config._repr_html_(title=f"DatasetConfig - <code>{name}</code>")
+                )
+
+        return Template("rendered_html_common.html.j2").render(content=content)
+
+    def _datasets_repr_html_(self) -> str:
+        content = []
+        if self.datasets:
+            for name, config in self.datasets.items():
+                content.append(
+                    Template("title_data.html.j2").render(
+                        title=f"DatasetConfig - <code>{name}</code>",
```

Comment:
Shouldn't this `Dataset`?

```suggestion
                        title=f"Dataset - <code>{name}</code>",
```

Reply:
Yep, nice catch :+1: